src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -281,6 +282,11 @@ ExpectedPrivateKey LoadFrom(const Args &args) {
 #endif // ndef MENDER_CRYPTO_OPENSSL_LEGACY
 
 ExpectedPrivateKey PrivateKey::Load(const Args &args) {
+	// Load OpenSSL config
+	if ((CONF_modules_load_file(NULL, NULL, 0) != 1)) {
+		log::Debug("Failed to load OpenSSL configuration file: " + GetOpenSSLErrorMessage());
+	}
+
```

Comment:
amazing find. thanks a lot. I only have some doubts about the place, Load method of the PrivateKey class does not seem a good place to load the config file.

Reply:
I agree, but I didn't find an obvious place to load it other than here, I can look a bit harder though :)

Reply:
Given the current design, I think `Load` is the best place, actually.

It is the entry point for creating a `PrivateKey` object from existing keys, and it is used from for the rest of the methods dealing with private keys like like `Sign` and `ExtractPublicKey`.